Dn-FamiTracker.reg

```diff
@@ -4,13 +4,18 @@ REGEDIT
 ;   registered in your InitInstance automatically with a call to
 ;   CWinApp::RegisterShellFileTypes and COleObjectFactory::UpdateRegistryAll.
 
-HKEY_CLASSES_ROOT\.0cc = "0CC-FamiTracker Module"
```

Comment:
I changed a portion of this line because the (Default)'string value for the file extension must correspond with the file type entry (key) in order for the file association to be successful. It should work fine now.